src/libcore/slice/mod.rs

```diff
@@ -2145,6 +2145,22 @@ impl<T> [T] {
         }
     }
 
+    /// Fills `self` with elements by cloning `value`.
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// let mut buf = vec![0; 10];
+    /// buf.fill(1);
+    /// assert_eq!(buf, vec![1; 10]);
+    /// ```
+    #[unstable(feature = "slice_fill", issue = "none")]
+    pub fn fill(&mut self, value: T) {
```

Comment:
```suggestion
    pub fn fill(&mut self, value: T) where T: Clone {
```

Reply:
Oops, good catch. I forgot to upload a patch for this; should be all fixed now!